resources/js/Pages/Contacts/Create.svelte

```diff
@@ -0,0 +1,107 @@
+<script>
+    import { Inertia } from '@inertiajs/inertia'
+    import { InertiaLink, page, remember } from '@inertiajs/inertia-svelte'
+    import { route } from '@/utils'
+    import Layout from '@/Shared/Layout.svelte'
+    import LoadingButton from '@/Shared/LoadingButton.svelte'
+    import SelectInput from '@/Shared/SelectInput.svelte'
+    import TextInput from '@/Shared/TextInput.svelte'
+
+    export let organizations = []
+
+    let sending = false
+    let form = remember({
+        first_name: null,
+        last_name: null,
+        organization_id: null,
+        email: null,
+        phone: null,
+        address: null,
+        city: null,
+        region: null,
+        country: null,
+        postal_code: null
+    })
+
+    function submit() {
+        sending = true
+        Inertia.post(route('contacts.store'), $form)
+            .then(() => sending = false)
+    }
+
+</script>
+
+<Layout title="Create Contact">
+    <h1 class="mb-8 font-bold text-3xl">
+        <InertiaLink href={route('contacts')} class="text-indigo-400 hover:text-indigo-600">
+            Contacts
+        </InertiaLink>
+        <span class="text-indigo-400 font-medium">/</span>
+        Create
+    </h1>
+    <div class="bg-white rounded shadow overflow-hidden max-w-3xl">
+        <form on:submit|preventDefault={submit}>
+            <div class="p-8 -mr-6 -mb-8 flex flex-wrap">
+                <TextInput
+                    bind:value={$form.first_name}
+                    errors={$page.errors.first_name}
+                    class="pr-6 pb-8 w-full lg:w-1/2"
+                    label="First name:" />
+                <TextInput
+                    bind:value={$form.last_name}
+                    errors={$page.errors.first_name}
+                    class="pr-6 pb-8 w-full lg:w-1/2"
+                    label="Last name:" />
+                <SelectInput
+                    bind:value={$form.organization_id}
+                    errors={$page.errors.organization_id}
+                    class="pr-6 pb-8 w-full lg:w-1/2"
+                    label="Organization:"
+                    let:selected>
```

Comment:
Since this is a data creation form and there's no data coming from the backend, it's safe to remove `let:selected` along with the select check below. Same for the `country` field.